src/main/java/fi/aalto/cs/intellij/notifications/BetaVersionWarning.java

```diff
@@ -18,7 +18,7 @@ public BetaVersionWarning(@NotNull Version version) {
     super("A+",
         "A+ Courses plugin is under development",
         "You are using version " + version + " of A+ Courses plugin, "
-            + "which is a pre-release version of the plugin and still under development. "
+            + "which is a pre-release version of the plugin and still under development. \n "
```

Comment:
Should platform independent line separator (`System.lineSeparator()`) used instead of LF? Does IntelliJ platform have any instructions on that?

Reply:
seems so, good point, I'll fix, thanks:
https://github.com/JetBrains/intellij-community/blob/e6abc7158fee3734d7ec74bbc53744798ea4bc78/platform/util/src/com/intellij/util/LineSeparator.java